packages/server/graphql/mutations/addFeatureFlag.ts

```diff
@@ -31,14 +34,27 @@ export default {
   },
   async resolve(
     _source: unknown,
-    {emails, domain, flag}: {emails: string[] | null; domain: string | null; flag: string},
+    {
+      emails,
+      domain,
+      flag
+    }: {emails: string[] | null; domain: string | null; flag: UserFeatureFlagEnum},
     {authToken, dataLoader}: GQLContext
   ) {
     const operationId = dataLoader.share()
     const subOptions = {operationId}
 
     // AUTH
-    requireSU(authToken)
+    const viewerId = getUserId(authToken)
+    const viewer = await getUserById(viewerId)
```

Comment:
superusers will never need to query the user.
come to think of it, same with regular users. the viewerId comes from a server-signed token, so the viewer is guarnateed to exist unless we hard deleted them & they're still using their old token.
even if that happened, the DB mutation below wouldn't fail, it'd just update 0 records